lambda-notes/src/Components/rightContent/MyNotes.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const MyNotes = (props) => {
+	return (
+		<div>
+			{props.notes.map((note) => {
+				return (
+					<NoteDetails key={note._id} title={note.title} textBody={note.textBody} _id={note._id} {...props}>
+						<Link to={`/viewNote/${note._id}`}> </Link>
```

Comment:
I don't think this Link is actually wrapped around anything clickable. It should be wrapped around `<NoteDetails />` so the card is clickable.